### DISCUSSION TITLE
Consolidate collection data from many collections in one

### DISCUSSION THREAD

**Discussion by shopline** [directus/directus#2845] [Archive]
I have 5 collections that I want to create a relationship in a combined collection.

Is this possible?

For example 5 collections with car brands:
Volvo
Audi
VW
BMW
Tesla

In these collections I have all the models of each brand.

In my final collection I want to list every single car model and their brand in a list from these collections where I can apply sorting.

**Comment by benhaynes** [comment]
This sounds like a "SQL View", which Directus does not  yet support. Though you could also build this as an extension (interface, module, layout, etc) to manually combine the data. Also, this will be far easier to do in v9 (on the `next` repo) and there's a good chance that SQL views will be supported soon (especially with the Feature Request upvoted).
